SerialPrograms/Source/PokemonSV/Programs/Eggs/PokemonSV_EggAutonomous.cpp

```diff
@@ -478,8 +493,6 @@ void EggAutonomous::process_one_baby(SingleSwitchProgramEnvironment& env, BotBas
 
         case EggHatchAction::Release:

```

Comment:
On phone so hard to check but is this enum value used anywhere else?

Reply:
I don't think so. It is the default action set on line 436.